### PR TITLE
feat: login with social works fine

### PR DESCRIPTION
**Pull request by felixdusengimana**
#### What does this PR do?
 
Added login with facebook and google functionalities

#### How should this be manually tested?

Check out this branch, run `npm install`, 
Make sure your .env have all properties in `.env.example`
then go to `{BASE_URL}/api/users/auth/google` or `{BASE_URL}/api/users/auth/facebook` and then you can login in with your credentials. 

#### Any background context you want to provide?

None

#### What are the relevant pivotal tracker stories?

# 184638374

#### Screenshots (if appropriate)

...

### CODE REVIEW

File: .env.example
Comment (oreste-abizera) [inline .env.example:1]: can you re-add EMAIL and APP_PASS variables in .env.example

**Review by dushimsam — APPROVED** [review]
LGTM

**Review by honorine22 — APPROVED** [review]

**Review by oreste-abizera — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+
 NODE_ENV=
-BASE_URL=http//:localhost:5001
 JWT_SECRET=
 JWT_EXPIRES_IN=2h
 APP_PASS="peeyhfnawmqcqphn"
 EMAIL="unflappablesbarefootnomad@gmail.com"
 
+# redirect urls
+BACKEND_URL=
```
Comment (Nkbtemmy) [inline .env.example]: Use BASE_URL instead of this; and update where you used BACKEND_URL

File: .env.example [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+FRONTEND_URL=
```
Comment (Nkbtemmy) [inline .env.example]: Also, this have been specified as WEB_URL

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**